governance/remote_executor/programs/remote-executor/src/tests/test_adversarial.rs

```diff
@@ -146,7 +145,7 @@ async fn test_adversarial() {
         .await
         .unwrap_err()
         .unwrap(),
-        InstructionError::PrivilegeEscalation.into_transation_error()
+        ErrorCode::ConstraintSeeds.into_transation_error()
```

Comment:
Change in anchor, the address of the PDA is checked before attempting to create the account